symphony/lib/toolkit/class.entrymanager.php

```diff
@@ -174,39 +174,48 @@ public static function edit(Entry $entry)
                 continue;
             }
 
+            $did_lock = false;
             try {
+                $did_lock = Symphony::Database()->query('LOCK TABLES tbl_entries_data_' . $field_id . ' WRITE');
                 Symphony::Database()->delete('tbl_entries_data_' . $field_id, sprintf("
```

Comment:
No, it means that no other session can access the table, which is exactly what we need. See http://dev.mysql.com/doc/refman/5.7/en/lock-tables.html